ui/pages/confirmations/components/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -59,14 +72,34 @@ function BlockaidBannerAlert({ txData, ...props }) {
 
   const t = useContext(I18nContext);
   const { updateTransactionEventFragment } = useTransactionEventFragment();
+  const [isLoadingDisplayed, setIsLoadingDisplayed] = useState(false);
+
+  useEffect(() => {
+    if (securityAlertResponse?.reason === 'loading') {
+      setIsLoadingDisplayed(true);
+    }
+  }, [securityAlertResponse]);
 
   if (
     !securityAlertResponse ||
     Object.keys(securityAlertResponse).length === 0
   ) {
     return null;
   } else if (securityAlertResponse.reason === 'loading') {
```

Comment:
should we update this to use `isLoadingDisplayed`? 
```suggestion
  } else if (isLoadingDisplayed) {
```